pom.xml

```diff
@@ -84,6 +84,10 @@
       <groupId>io.jenkins.plugins</groupId>
       <artifactId>caffeine-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>gitlab-api</artifactId>
+    </dependency>
```

Comment:
In addition to the addition of this dependency we also need to remove RESTEasy and any imports of `javax.ws.rs.*`. Otherwise we retain dead code that is no longer needed as well as a potential source of security vulnerabilities.